_data/people/DmitryBocharov.md

```diff
@@ -7,4 +7,4 @@ location: Moscow, Russia
 occupation: 
 gravatar_hash: b1e76c4f863625690751dc0d4b89ac1a
 ---
-Dmitry was born in Kaluga, Russia and graduated with a diploma in applied mathematics and physics from the Moscow Institute of Physics and Technology. He became interested in Java since the first university years of the university. Worked for several companies in Moscow, developing CRM systems for a large telecommunication company and creating Hadoop-based ETL processes for an international bank. In 2015 he participated in the GSOC program and became a fan of open source, contributing to Hibernate Search, JBoss Tools and Eclipse JSDT.
+Dmitry was born in Kaluga, Russia and graduated with a diploma in applied mathematics and physics from the Moscow Institute of Physics and Technology. He became interested in Java since the first university years of the university. Worked for several companies in Moscow, developing CRM systems for a large telecommunication company and creating Hadoop-based ETL processes for an international bank. In 2015 he participated in the GSOC program and became a fan of open source, contributing to Hibernate Search, JBoss Tools and Eclipse JSDT. Now he is working in Red Hat on JBoss Tools plugins for Openhift and Hibernate.
```

Comment:
s/Openhift/Openshift/

And I think it should be "working for Red Hat" rather than "working in Red Hat"